content/navigate-history.en.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Browsing through terms"
+weight: 10
+---
+
+# Browsing through terms
+
+Every collection offers a public database of versions they recorded. 
+
+For this guide, we will use the example of the demo collection. This collection publishes its history on the [`OpenTermsArchive/demo-versions`](https://github.com/OpenTermsArchive/demo-versions) repository.
+
+- From the [repository page](https://github.com/OpenTermsArchive/demo-versions), open the folder of the service of your choice by clicking on it. For example, [GitHub](https://github.com/OpenTermsArchive/demo-versions/tree/main/GitHub):
+
+  ![Demo-versions repository services list](/images/navigate-history/demo-repository.png)
+
+- You will see the set of documents tracked for that service, now click on the document of your choice. For example, [GitHub's Privacy Policy](https://github.com/OpenTermsArchive/demo-versions/blob/main/GitHub/Privacy%20Policy.md):
+
+  ![GitHub terms list](/images/navigate-history/github-terms.png)
+
+- The latest version will be displayed. To view the history of changes made to this document, click on _History_ at the top right of the document:
```

Comment:
> This is correct, but let's make it even less ambiguous 🙂 

```suggestion
- The most recent version will be displayed. To view the history of changes made to these terms, click on _History_ at the top right:
```

We could also use Primer Octicon's [history icon](https://primer.style/design/foundations/icons/history-24) since it's coming from GitHub.